packages/widgetbook/test/src/addons/custom_theme_addon_test.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widgetbook/src/addons/addons.dart';
+import 'package:widgetbook/src/addons/theme_addon/theme_selection_provider.dart';
+import 'package:widgetbook/src/workbench/renderer.dart';
+
+import 'utils/custom_app_theme.dart';
+import 'utils/widget_wrapper.dart';
+
+void main() {
+  late Renderer sut;
+  final rendererKey = GlobalKey();
+
+  setUpAll(() {
+    sut = Renderer(
+      key: rendererKey,
+      appBuilder: (context, child) {
+        final frameBuilder = context.frameBuilder;
+        final theme = context.theme<AppThemeData>();
+        return AppTheme(
+          data: theme,
+          child: frameBuilder.builder(
+            context,
+            child,
+          ),
+        );
+      },
+      useCaseBuilder: (context) => Scaffold(
+        backgroundColor: AppTheme.of(context).color,
+        body: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: const [
+              Text(
+                'This is the home page',
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  });
+
+  group('$Renderer', () {
+    testWidgets(
+      'renders correctly',
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          addOnProviderWrapper(
+            child: sut,
+          ),
+        );
+
+        expect(find.byKey(rendererKey), findsOneWidget);
+      },
+    );
+    testWidgets(
+      'renders one Text Widget',
```

Comment:
Also unrelated to the addon. This is pretty much the same test as the one above.